### PR TITLE
Issue #353 Adding tip for using 'glide cc'

### PR DESCRIPTION
**Pull request by hferentschik**
- Formatting and style alignment

Addresses issue #353

### CODE REVIEW

File: docs/source/developing/developing.adoc [hunk 1]
```diff
@@ -115,17 +113,19 @@ $ glide install -v
...
+ Glide will recognize that there is no lock file and recalculate the required dependencies.
 
-. Check-in the updated _glide.lock_ file and the updated _glide.yaml_.
+. Check-in the updated *_glide.yaml_* and *_glide.lock_*.
 
-. Test that everything still compiles with the new lock file in place.
+. Test that everything still compiles with the new lock file in place: `make clean && make`.
+
+TIP: In some cases the Glide cache located under *_~/.glide/cache_* can get corrupted.
+If you seeing Glide errors during `make vendor`, you can clear the Glide cache via `glide cc`.
```
Comment (praveenkumar) [inline docs/source/developing/developing.adoc]: 👍

Comment (hferentschik) [inline docs/source/developing/developing.adoc]: :-)

**Comment by hferentschik** [comment]
Can we merge this?

File: docs/source/developing/developing.adoc [hunk 2]
```diff
@@ -18,9 +18,8 @@ The following sections describe how to build and test Minishift.
 - Git
 - A recent Go distribution (>1.7)
 
-NOTE: You should be able to develop Minishift on any operating system, such as GNU/Linux,
-Mac OS X or Windows. The Windows operating system might require additional steps or
-have some limitations.
+NOTE: You should be able to develop Minishift on any operating system, such as GNU/Linux, Mac OS X or Windows.
```
Comment (budhrg) [inline docs/source/developing/developing.adoc]: `Mac OS X` -> `macOS` Refer https://en.wikipedia.org/wiki/Macintosh_operating_systems.

Comment (hferentschik) [inline docs/source/developing/developing.adoc]: Done

File: docs/source/developing/developing.adoc [hunk 3]
```diff
@@ -71,10 +69,9 @@ $ git clone https://github.com/minishift/minishift.git github.com/minishift/mini
 [[using-ide]]
 === Using an IDE
 
-You can use any editor you want. However, most of the core maintainers of Minishift use Intellij
-https://www.jetbrains.com/idea/[IDEA] with the latest Go plugin. This IDE indexes your
-whole workspace and allows for easy navigation of the sources, and also integrates
-with the Go debugger https://github.com/derekparker/delve[Delve].
+You can use any editor you want.
+However, most of the core maintainers of Minishift use Intellij https://www.jetbrains.com/idea/[IDEA] with the latest Go plugin.
```
Comment (budhrg) [inline docs/source/developing/developing.adoc]: @hferentschik , please change this `Intellij IDEA` to `IntelliJ IDEA`.

**Comment by budhrg** [comment]
Merged and thanks!

**Pull request closed without merging**